cm-mlops/script/get-generic-python-lib/_cm.json

```diff
@@ -516,6 +516,14 @@
         "CM_SCIKIT_LEARN_VERSION"
       ]
     },
+    "cupy": {
```

Comment:
cupy has a dependency on CUDA right?

Reply:
Yes its for running numpy and scipy codes in CUDA

Reply:
Can you please add that as a deps for cupy?

Reply:
Sure